Code/Engine/Foundation/Memory/Implementation/MemoryUtils_inl.h

```diff
@@ -377,19 +377,19 @@ EZ_ALWAYS_INLINE void ezMemoryUtils::Destruct(T* pDestination, size_t uiCount, e
 }
 
 template <typename T>
-EZ_ALWAYS_INLINE void ezMemoryUtils::Destruct(T* pDestination, size_t uiCount, ezTypeIsClass)
+void ezMemoryUtils::Destruct(T* pDestination, size_t uiCount, ezTypeIsClass)
 {
   EZ_CHECK_CLASS(T);
 
-#define EZ_GCC_WARNING_NAME "-Waggressive-loop-optimizations"
-#include <Foundation/Basics/Compiler/GCC/DisableWarning_GCC.h>
+  EZ_WARNING_PUSH()
+  // EZ_WARNING_DISABLE_GCC("-Waggressive-loop-optimizations")
```

Comment:
why is this commented out?

Reply:
This was a test (successful!). The warning push/pop etc can be removed now.